pkg/influx/query/query.go

```diff
@@ -0,0 +1,48 @@
+package query
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/valyala/fasthttp"
+)
+
+func GetQueryHandler(ctx *fasthttp.RequestCtx, myid uint64) {
+
+	q := string(ctx.QueryArgs().Peek("q"))
+	if q == "show databases" {
+		databaseResponse := `{
+			"results": [
+				{
+					"series": [
+						{
+							"name": "databases",
+							"columns": ["name"],
+							"values": [
+								["_internal"],
+								["benchmark_db"]
+							]
+						}
+					]
+				}
+			]
+		}`
+		ctx.SetContentType("application/json")
+		_, err := ctx.Write([]byte(databaseResponse))
+		if err != nil {
+			log.Errorf("GetQueryHandler: failed to write response, err=%v", err)
+		}
+	} else if q == "CREATE DATABASE benchmark WITH REPLICATION 1" {
```

Comment:
we should just check if `q.contains("CREATE DATABASE")` 
we don't care about what the dbName is and we should just return 200 for this

Reply:
I've pushed the changes for this to check if the query contains `create database`. Do we want to return an error if it just contains `create database` without a database name?